### ISSUE TITLE
Add screenshots to admin docs

### ISSUE THREAD

**Issue by benjaminjkraft** [learning-unlimited/ESP-Website#2058]
According to @milescalabresi and @bhorowitz, chapter admins have said that having more screenshots in admin documentation would make it more useful and easier to understand.  So we may want to put some effort into having more screenshots in our documentation, especially the admin docs.

**Comment by milescalabresi** [comment]
I have one from last year's push complete with screenshots.
[How to send an automatic confirmation email.pdf](https://github.com/learning-unlimited/ESP-Website/files/150212/How.to.send.an.automatic.confirmation.email.pdf)

**Comment by benjaminjkraft** [comment]
Thanks @milescalabresi -- any interest in converting that into a doc in the repo?  Especially if you have the source it shouldn't be hard, and that way it will be easy for us to find and update.

Some ideas for things we might want to have screenshots of (feel free to edit and add to this list):
* [ ] themes
* [ ] Ajax scheduler
* [ ] Setting up a program
* [ ] Student and teacher registration
* [ ] Dashboard and studentreg bigboard

**Comment by milescalabresi** [comment]
Yes, I can do that. Daniel Hwang and I were working on a more complete set of documents that could serve as a beginner's guide as well as a reference. It's text-only right now, but we were hoping release it in stages and finish the first stage (complete with pictures and all) within the next month. I'll add those to the website as well when I get a chance. I'll make sure we cover those ideas.

**Comment by pushkarkumar06** [comment]
Hi @willgearty ! 👋
I’d love to work on this issue #2058 and help improve the admin documentation by adding useful screenshots. If it’s okay, please assign this issue to me. I’m happy to follow the project guidelines and get started right away.
Thanks!

**Comment by willgearty** [comment]
@pushkarkumar06 Thanks for your interest in contributing to our project! I've assigned you to this issue. Please make sure to check out our [instructions for getting started](https://github.com/learning-unlimited/ESP-Website/wiki#getting-started) and follow our [contribution rules](https://github.com/learning-unlimited/ESP-Website/wiki#rules). Let me know if you have any questions.

**Comment by pushkarkumar06** [comment]
Hi @willgearty, I've been assigned to issue #2058 (Add screenshots to admin docs). Before I start, I wanted to confirm a few things:

1. Screenshot source — Should the screenshots be taken from a locally running dev server, or do you have a staging/production server I should use for more realistic looking screenshots?

2. Screenshot scope — The issue mentions themes, scheduler, setting up a program, program modules, student/teacher registration, dashboard and studentreg bigboard, and site configuration. Should I focus on all of these, or prioritize specific ones for this PR?

3. Format/location — Should images go in [images](vscode-file://vscode-app/usr/share/code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) as .png files embedded via RST .. figure:: directives (following the existing pattern in available_themes.rst)? Or is there a preferred format/location?

4. No active program — My local dev setup only has the bare vagrant user with no programs created yet. Should I create a test program locally to take screenshots of the scheduler/modules pages, or will you provide sample screenshots?

Thanks for your guidance!

**Comment by willgearty** [comment]
1. We do have a staging server, but it has private data on it, so we can't share that with you. Same thing with our production server. So you'll probably need to make a realistic looking dev environment and then take the screenshots there.
2. It's up to you, you can break it down into smaller pieces or do it all as one PR.
3. The files should be PNDs and they should in https://github.com/learning-unlimited/ESP-Website/tree/main/docs/admin/images
4. You can use my dev database dump, if you'd like: [dumped_db.sql](https://github.com/user-attachments/files/25507736/dumped_db.sql). It's a bit messy, and you'll still need to add some things like logos.

**Comment by pushkarkumar06** [comment]
Hi @willgearty , 
I'm working on the admin docs screenshots issue. I've loaded the database dump successfully and can see 10 programs at /manage/programs/. However I can't find the correct URL format to access the program management page. Could you share the correct URL format to access the management panel for a specific program like Splash/2023_Test2? 
Thank you!

**Comment by pushkarkumar06** [comment]
Hi @willgearty ,

I've successfully completed this issue and submitted **PR #4375**: https://github.com/learning-unlimited/ESP Website/pull/4375

### What was done:
- Added 4 screenshots to the admin documentation:
  - `themes_main.png` - Themes management interface
  - `dashboard.png` - AdminCore program dashboard  
  - `ajax_scheduler.png` - AJAX scheduling interface
  - `bigboard.png` - BigBoard statistics page
- Updated `themes.rst` to include the themes screenshot
- Updated `program_modules.rst` to include screenshots for AdminCore, AJAX Scheduler, and BigBoard sections

### Setup process:
I loaded your database dump (`dumped_db.sql`) into the local dev environment, configured the program with all modules, and took screenshots of the working interfaces.

Here's a preview of one of the added screenshots:

![Dashboard Screenshot](https://github.com/pushkarkumar06/ESP-Website/blob/add-screenshots-admin-docs-2058/docs/admin/images/dashboard.png?raw=true)

Please let me know if any improvements are needed or if you'd like me to add screenshots for any other sections.

Thank you!

**Comment by willgearty** [comment]
Looks like you figured out the URLs 👍 Let me know if you have any other questions.

**Comment by pushkarkumar06** [comment]
Hi @willgearty , thanks for the feedback on my PR #4375!

1. Branch is now cleaned up — it contains only the docs/screenshots commit (1 commit, 6 files).

2. Regarding the screenshot quality — I'll wait for #3643 (New dummy database) to finish and retake the ajax_scheduler and bigboard screenshots once there's realistic data to work with. 
3. Let me know if you'd prefer a different approach.

Thank you !

**Comment by willgearty** [comment]
@pushkarkumar06 please respond on the PR discussion so we don't have to pop back and forth between this issue and the PR
